templates/gerp-elements.yaml

```diff
@@ -0,0 +1,20 @@
+genome_id: <genome_id>
+category:
+  track_category_id: assembly
+  label: Assembly
+  type: Genomic
+trigger:
+  - track
+  - expand
+type: regular
+label: Constrained elements
+display_order: 960
+on_by_default: true
```

Comment:
Was there any decision about whether all of these tracks should be on by default?

_(I don't remember whether the client respects this flag; but that's another matter.)_

Reply:
This is why we have templates here: we (or data teams or product owner etc) can review and edit properties like `on_by_default` and it will be used the next time data loading script is run. Whether the constrained elements track should to be on by default, I don't know. Who do we ask?